dev/fattest.simplicity/src/componenttest/topology/utils/ExternalTestServiceDockerClientStrategy.java

```diff
@@ -123,7 +123,7 @@ public boolean isMatched(ExternalTestService dockerService) {
 
         public void test() throws InvalidConfigurationException {
             final String m = "test";
-            final int maxAttempts = FATRunner.FAT_TEST_LOCALRUN ? 1 : 7; // attempt up to 7 times for remote builds
+            final int maxAttempts = FATRunner.FAT_TEST_LOCALRUN ? 1 : 4; // attempt up to 4 times for remote builds
```

Comment:
1 min timeout feels like it might be a tad agressive to overcome infrastructure, happy to change this for now but would recommend revisiting if we get frequent timeouts